Babylon/utils/environment.py

```diff
@@ -1,26 +1,45 @@
 import os
```

Comment:
I thought you would use a module variable
```python
class Environment():
    pass

environment = Environment()
```
and then 
```python
from ..utils.environment import environment
```
But a real singleton is another option !

Reply:
I never did a Singleton before so it was an occasion to do it ^^